### PR TITLE
[new] Add react-router-redux v4.0.4 with npm auto-update config

### PR DESCRIPTION
**Pull request by Dattaya**
# Profile of the lib
 * Git repository (required): https://github.com/reactjs/react-router-redux
 * Official website (optional, not the repository): -
 * NPM package url (optional): https://www.npmjs.com/package/react-router-redux
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 3, 053
   - Count of watchers: 118
   - Count of forks: 238
 * NPM download stats (optional):
   - Downloads in the last day: 2, 216
   - Downloads in the last week: 39,155
   - Downloads in the last month: 195,073

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid. (I hope)
 * [x] Auto-update filemap is correct. (same as above)

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

I'm trying to follow @PeterDaveHello's advice from #3941:
> @ clayreimann we'll need much more time on the discussion about how to handle the files, but at the mean time, actually you can submit a PR on GitHub with only a single package.json, please take a look at #7149

This is my first time sending a library to `cdnjs`, so I could've made a mistake, sorry.

### CODE REVIEW

File: ajax/libs/react-router-redux/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/reactjs/react-router-redux.git"
+  },
+  "authors": [
+    "James Long",
+    "Tim Dorr"
+  ],
+  "license": "MIT",
+  "tags": [
```
Comment (PeterDaveHello) [inline ajax/libs/react-router-redux/package.json]: Would you please remove "tags" section, we don't use this, thanks.

File: ajax/libs/react-router-redux/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "react",
+    "redux",
+    "router"
+  ],
+  "npmName": "react-router-redux",
+  "npmFileMap": [
+    {
+      "basePath": "/dist/",
+      "files": [
+        "*.js"
```
Comment (PeterDaveHello) [inline ajax/libs/react-router-redux/package.json]: What about `"**/*"` to include all the files recursively?

File: ajax/libs/react-router-redux/package.json (hunk 1 above)
Comment (Dattaya) [inline ajax/libs/react-router-redux/package.json]: done

File: ajax/libs/react-router-redux/package.json (hunk 2 above)
Comment (Dattaya) [inline ajax/libs/react-router-redux/package.json]: & done

**Comment by Dattaya** [comment]
@PeterDaveHello, thank you for reviewing this pull request. I had to do sparse checkout after all (to be able to squash the commits), thankfully, without any trouble. 
Please, let me know if there are other mistakes.

**Comment by PeterDaveHello** [comment]
@Dattaya in fact, you don't need to, you can just edit the file on corresponding branch on GitHub.

**Comment by Dattaya** [comment]
>  you can just edit the file on corresponding branch on GitHub.
Thanks, good to know.

**Comment by PeterDaveHello** [comment]
np.

Now you can see it on:
https://github.com/cdnjs/cdnjs/commit/902b8e44d1e3805867fbb0b64c7b93d25c31be9c
https://cdnjs.com/libraries/react-router-redux

**Pull request merged**